src/server/hset_family.cc

```diff
@@ -1201,4 +1204,25 @@ StringMap* HSetFamily::ConvertToStrMap(uint8_t* lp) {
 
   return sm;
 }
+
+// returns -1 if no expiry is associated with the field, -3 if no field is found.
```

Comment:
very small nit: better we returned an optional<uint32_t> here (field not found), and did the casting to -3 in the command handler itself

Reply:
why it is better on your opinion? we need to return multiple statuses: -3, -1.